tests/test_envs.py

```diff
@@ -163,6 +163,13 @@ def test_max_steps_argument(env_spec):
     env.close()
 
 
+@pytest.mark.parametrize("env_id", [spec.id for spec in all_testing_env_specs])
+def test_env_picklable(env_id):
```

Comment:
Can you revert to your test implementation with pickle dumps and load?

Reply:
Isn't this just update on not being done yet ? It would be strange that it require an other commit